bool ready # to tell the analysis node ready or not
int32 scanning_count # No. of scan
